docs/modules/getting-started/pages/get-started-rest-api-with-docker.adoc

```diff
@@ -0,0 +1,182 @@
+                        <user username="restuser" password="restpassword">
+                            <role>admin</role>
+                        </user>
+                    </simple>
+                </authentication>
+                <access-control-service>
+                    <factory-class-name>com.hazelcast.internal.rest.access.DefaultAccessControlServiceFactory</factory-class-name>
+                </access-control-service>
+            </realm>
+        </realms>
+    </security>
+</hazelcast>
+----
+--
+
+YAML::
++
+[source,yaml]
+----
+hazelcast:
+  license-key: <your license key> <1>
+  rest:
+    enabled: true
+    security-realm: restRealm
+  security:
+    enabled: true
+    realms:
+      - name: restRealm
+        authentication:
+          simple:
+            users:
+              - username: 'restuser'
+                password: 'restpassword'
+                roles:
+                  - admin
+        access-control-service:
+          factory-class-name: com.hazelcast.internal.rest.access.DefaultAccessControlServiceFactory
+----
+====
+<1> Replace the `<your license key>` placeholder with your Hazelcast {enterprise-product-name} license key.
+
+== Step 2. Start Hazelcast {enterprise-product-name} Server
+
+To start the Hazelcast {enterprise-product-name} server, run the following command:
+
+If you have XML config file:
+[source,shell,subs="attributes+"]
+----
+docker run \
+    -p 5701:5701 \
+    -e JAVA_OPTS="-Dhazelcast.config=/opt/hazelcast/config_ext/hazelcast.xml" -v ~/config:/opt/hazelcast/config_ext hazelcast/hazelcast-enterprise:{full-version}
+----
+
+If you have YAML config file:
+[source,shell,subs="attributes+"]
+----
+docker run \
+    -p 5701:5701 \
+    -e JAVA_OPTS="-Dhazelcast.config=/opt/hazelcast/config_ext/hazelcast.yaml" -v ~/config:/opt/hazelcast/config_ext hazelcast/hazelcast-enterprise:{full-version}
+----
+
+This command starts the member and configures it, using your configuration file.
+Here, `~/config` is the path to the directory in which you saved the configuration file.
+If you saved the configuration file somewhere else, you should replace this path.
+
+You should see a message in the console indicating that the REST service is enabled, similar to the following:
+
+[source,shell,subs="attributes+"]
+----
+com.hazelcast.internal.rest.init.RestServiceImpl
+INFO: [192.168.0.24]:5701 [dev] [5.5.0-SNAPSHOT] Hazelcast REST Service is enabled on port: 8443 with security realm: restRealm and access control service: com.hazelcast.internal.rest.access.DefaultAccessControlService
+----
+
+== Step 3. Access the REST API and Swagger UI
+
+The REST API is running on port 8443.
+You can access the Swagger UI at: http://localhost:8443/swagger-ui/index.html
+
+== Step 4. Obtain a Token to Access All Endpoints
+To obtain a token, send a POST request to the endpoint at /hazelcast/rest/api/v1/token.
+
+*Using Swagger UI:*
+
+- Open the Swagger UI at http://localhost:8443/swagger-ui/index.html.
```

Comment:
Make 2 tabs, one Swagger UI, 2nd is the Curl way. Do it for this and the next step.

Reply:
fixed at [efcd8b0](https://github.com/hazelcast/hz-docs/pull/1136/commits/efcd8b002c471657940150e14c777febc33a9f1b)